### PR TITLE
Lets admins grant any objective instead of traitor only ones + crew obj bugfixes

### PR DESCRIPTION
**Pull request by Carbonhell**
- tweak: "Admins can now give any objective and set the target easily."
- bugfix: "Fixed latejoiner antags having crew objectives."
- bugfix: "Fixed the have x chemical inside at roundend objective not actually requiring any chem."

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -295,13 +295,19 @@
...
-				ticker.mode.make_antag_chance(character)
+				if(ticker.mode.make_antag_chance(character))
+					antag = TRUE
 			if(SHUTTLE_CALL)
 				if(SSshuttle.emergency.timeLeft(1) > initial(SSshuttle.emergencyCallTime)*0.5)
-					ticker.mode.make_antag_chance(character)
+					if(ticker.mode.make_antag_chance(character))
+						antag = TRUE
+	if(!antag && character.mind)
+		world << "fuck"
```
Comment (ghost) [inline code/modules/mob/new_player/new_player.dm]: ???

**Comment by ghost** [comment]
But yeah okay +1 nice feature my dudes.

**Comment by Carbonhell** [comment]
fixed the last things,should be good now

**Comment by Lawligagger** [comment]
ok

**Comment by Kokojos** [comment]
Playtesting made this codechecked ++

Also +0.33

**Pull request merged**